src/modal.test.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import TestUtils from 'react-addons-test-utils';
+import Modal from './Modal';
+import Icon from './Icon';
+import Button from './Button';
+
+describe('Modal', () => {
+
+	let modal, modalEl, spyable;
+	const content = 'Test model content';
+
+	beforeEach(() => {
+		spyable = {
+			onDismiss: (e) => {}
+		};
+
+		spyOn(spyable, 'onDismiss');
+
+		modal = TestUtils.renderIntoDocument(
+			<Modal onDismiss={spyable.onDismiss}>{content}</Modal>
+		);
+		modalEl = ReactDOM.findDOMNode(modal);
+	});
+
+	afterEach(() => {
+		modalEl = null;
+	});
+
+	it('exists', () => {
+		expect(modalEl).not.toBeNull();
+	});
+
+	it('has SQ2 modal styles', () => {
+		expect(modalEl.classList).toContain('modal');
+	});
+
+	it('displays modal content', () => {
+		expect(modalEl.innerHTML).toContain(content);
+	});
+
+	it('creates a Button component for dismissal', () => {
+		const len = TestUtils.scryRenderedComponentsWithType(modal, Button).length;
+		expect(len).toBeGreaterThan(0);
```

Comment:
this is another case where you could give the `dismiss` button a classname and search for that instead of looking for an anonymous `Button` somewhere in the modal - this test is finding more than just the `dismiss` button, so it would pass even if the dismiss button was gone.